view/frontend/templates/css/boltcss.phtml

```diff
@@ -36,11 +32,6 @@ if ($block->shouldDisableBoltCheckout()) return;
         border: 1px solid #cccccc;
         width:200px;
     }
```

Comment:
@njfaries the above section can be added to Global CSS configuration defaults and removed from here as well.
https://github.com/BoltApp/bolt-magento2/blob/master/etc/config.xml#L47